frontend/src/config/somalia/layers.json

```diff
@@ -0,0 +1,288 @@
+        {
+          "key": "rfh_avg",
+          "label": "Average",
+          "color": "#bdf2e6"
+        }
+      ],
+      "levels": [
+        {
+          "level": "1",
+          "id": "dv_adm1_id",
+          "name": "ADM1_EN"
+        },
+        {
+          "level": "2",
+          "id": "dv_adm2_id",
+          "name": "ADM2_EN"
+        }
+      ],
+      "type": "bar"
+    }
+  },
+  "rain_anomaly_dekad": {
+    "chart_data": {
+      "fields": [
+        {
+          "key": "rfq",
+          "label": "Rainfall anomaly",
+          "color": "#375692",
+          "minValue": 0,
+          "maxValue": 300
+        },
+        {
+          "key": "rfq_avg",
+          "label": "Normal",
+          "fallback": 100,
+          "pointRadius": 0,
+          "color": "#eb3223"
+        }
+      ],
+      "levels": [
+        {
+          "level": "1",
+          "id": "dv_adm1_id",
+          "name": "ADM1_EN"
+        },
+        {
+          "level": "2",
+          "id": "dv_adm2_id",
+          "name": "ADM2_EN"
+        }
+      ],
+      "type": "line"
+    }
+  },
+  "rainfall_agg_1month": {
+    "chart_data": {
+      "fields": [
+        {
+          "key": "r1h",
+          "label": "Rainfall",
+          "color": "#233f5f"
+        },
+        {
+          "key": "r1h_avg",
+          "label": "Average",
+          "color": "#bdf2e6"
+        }
+      ],
+      "levels": [
+        {
+          "level": "1",
+          "id": "dv_adm1_id",
+          "name": "ADM1_EN"
+        },
+        {
+          "level": "2",
+          "id": "dv_adm2_id",
+          "name": "ADM2_EN"
+        }
+      ],
+      "type": "bar"
+    }
+  },
+  "rain_anomaly_1month": {
+    "chart_data": {
+      "fields": [
+        {
+          "key": "r1q",
+          "label": "Rainfall anomaly",
+          "minValue": 0,
+          "maxValue": 300,
+          "color": "#375692"
+        },
+        {
+          "key": "rfq_avg",
+          "label": "Normal",
+          "fallback": 100,
+          "pointRadius": 0,
+          "color": "#eb3223"
+        }
+      ],
+      "levels": [
+        {
+          "level": "1",
+          "id": "dv_adm1_id",
+          "name": "ADM1_EN"
+        },
+        {
+          "level": "2",
+          "id": "dv_adm2_id",
+          "name": "ADM2_EN"
+        }
+      ],
+      "type": "line"
+    }
+  },
+  "rainfall_agg_3month": {
+    "chart_data": {
+      "fields": [
+        {
+          "key": "r3h",
+          "label": "Rainfall",
+          "color": "#233f5f"
+        },
+        {
+          "key": "r3h_avg",
+          "label": "Average",
+          "color": "#bdf2e6"
+        }
+      ],
+      "levels": [
+        {
+          "level": "1",
+          "id": "dv_adm1_id",
+          "name": "ADM1_EN"
+        },
+        {
+          "level": "2",
+          "id": "dv_adm2_id",
+          "name": "ADM2_EN"
+        }
+      ],
+      "type": "bar"
+    }
+  },
+  "rain_anomaly_3month": {
+    "chart_data": {
```

Comment:
is the chart data different than for the other countries? could this be pulled from `shared` ?

Reply:
I'll revisit charts across the board soon. As of now, the variable name isn't standardized